roles/nis-client/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+nis_client_use_nscd: yes
```

Comment:
Should this be enabled by default? `nscd` is often an operational headache, especially on large clusters where the caches can get out of sync. I'd argue that the user should have to explicitly turn this on and know what they're getting into. :)